spec/models/subscription_spec.rb

```diff
@@ -149,4 +149,23 @@
       it { is_expected.to be_empty }
     end
   end
+
+  describe ".with_paid_transactions" do
+    let(:subscription_with_paid_transactions) { create(:subscription, :canceled) }
```

Comment:
Does the `canceled` state matter in this case?
The same question for `pending_payment`.